core/templates/dev/head/pages/exploration_editor/settings_tab/SettingsTab.js

```diff
@@ -53,7 +53,7 @@ oppia.controller('SettingsTab', [
 
     $scope.TAG_REGEX = GLOBALS.TAG_REGEX;
 
-    var DASHBOARD_PAGE_URL = '/dashboard';
+    var DASHBOARD_PAGE_URL = '/creator_dashboard';
```

Comment:
CREATOR_DASHBOARD_PAGE_URL?

(May be worth grepping for other instances of "DASHBOARD" too, just in case.)